spec/services/checksum_validator_spec.rb

```diff
@@ -3,19 +3,19 @@
 
 RSpec.describe ChecksumValidator do
   include_context 'fixture moabs in db'
+  let(:druid) { 'zz102hs9687' } # most common druid
```

Comment:
"most common druid" is apparently a "note of interest" but not actually meaningful.  Tee hee.

Oh wait - I see -- this is trying to say WHY you put zz102hs9687 here.  I would omit the comment, but it's not  important.

Reply:
i'll remove it! - repushed now